src/main/java/com/quirko/app/GameController.java

```diff
@@ -10,7 +10,7 @@
 
     private Board board = new SimpleBoard(25, 10);
 
-    private final GuiController viewGuiController;
+    public static  GuiController viewGuiController;

```

Comment:
Instead of defining this as public, implement an accessor method and call that method.